### PR TITLE
'Color Coded Tilemap' doc fixup

### PR DESCRIPTION
**Pull request by ganicke**
Move the related legacy tilemap reference pages into the `color-coded-tilempap` doc space. Include a top-level page to satisfy the extension's "Learn More" destination link.

Closes https://github.com/microsoft/pxt-arcade/issues/5581

### CODE REVIEW

File: libs/color-coded-tilemap/docs/reference/color-coded-tilemap/place-on-random-tile.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# place On Random Tile
+
+Move a sprite's position to the center of a random tile in the scene.
+
+```sig
+scene.placeOnRandomTile(null, 0)
+```
+
+You a can make a sprite to locate itself right on top of a random tile in the tile map. Use the color number of type of tile to select for the sprite to locate on one of them.
```
Comment (kimprice) [inline libs/color-coded-tilemap/docs/reference/color-coded-tilemap/place-on-random-tile.md:9]:
```suggestion
You can make a sprite locate itself right on top of a random tile in the tile map. Use the color number of type tile to select for the sprite to locate on one of them.
```

File: libs/color-coded-tilemap/docs/reference/color-coded-tilemap/place.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Move a sprite's position to the center of a tile in the scene.
 scene.getTile(0, 0).place(null)
 ```
 
-You a can make a sprite to locate itself right on top of a tile. If you have a [tile](/types/tile) object from the tilemap, just use **place** to put a sprite on top of it.
+You a can make a sprite to locate itself right on top of a tile. If you have a [tile](/reference/color-coded-tilemap/tile) object from the tilemap, just use **place** to put a sprite on top of it.
```
Comment (kimprice) [inline libs/color-coded-tilemap/docs/reference/color-coded-tilemap/place.md:9]:
```suggestion
You can make a sprite locate itself right on top of a tile. If you have a [tile](/reference/color-coded-tilemap/tile) object from the tilemap, just use **place** to put a sprite on top of it.
```

File: libs/color-coded-tilemap/docs/reference/color-coded-tilemap/set-tile-at.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
+# set Tile At
+
+Change a tile in the tilemap to another tile.
+
+```sig
+scene.setTileAt(null, 0)
+```
+
+A tile from the tile map has the [tile](/reference/color-coded-tilemap/tile) type. You can change that tile by using it's object type, [tile](/reference/color-coded-tilemap/tile), and the index of a different tile. The tile map is changed to have the new tile take the position in the tilemap where this one was.
```
Comment (kimprice) [inline libs/color-coded-tilemap/docs/reference/color-coded-tilemap/set-tile-at.md:9]:
```suggestion
A tile from the tile map has the [tile](/reference/color-coded-tilemap/tile) type. You can change that tile by using its object type, [tile](/reference/color-coded-tilemap/tile), and the index of a different tile. The tile map is changed to have the new tile take the position in the tilemap where this one was.
```

**Review by kimprice — APPROVED** [review]
Looks good other than a few typos!

**Comment by ganicke** [comment]
Wow, these were typos from more than two years ago.

**Pull request merged**